spec/mailers/user_sign_in_mailer_spec.rb

```diff
@@ -2,19 +2,30 @@
 
 RSpec.describe UserSignInMailer, type: :mailer do
   describe 'send invite email' do
+
+    let(:template) { '31d2b530-8db4-44ce-9bfa-34b01519335f' }
```

Comment:
Ideally we'd use a dummy value like 'example-template' or pull the default value from councils.yml, because if we change the value in councils.yml later, this test will fail. But not a big deal.